src/components/layout.js

```diff
@@ -90,17 +81,15 @@ const Layout = ({ location, title, children }) => {
         <span>
           © {new Date().getFullYear()} Excalidraw
           {" • "}
-          <a href="https://github.com/excalidraw/excalidraw-blog/blob/master/LICENSE">
-            MIT Licensed
-          </a>
```

Comment:
I think we want to keep the license here so people know how they can use the code or text in the post. Otherwise I believe the default is “All rights reserved”

Reply:
I think people will copy regardless, and we are safe, since we are all about open source anyways.. I wanted to clean it up and remove the links that not so important.

Reply:
I think the license is more important than a link to the blog homepage, since there’s always a link to the homepage at the top of the page. cc @excalidraw/blog for thoughts?

Reply:
I just never seen it, nor think that anyone cares about it..

Reply:
Added it back..